ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -770,6 +770,18 @@ static TimeDuration slownessTimeout(RaftProperties properties) {
     static void setSlownessTimeout(RaftProperties properties, TimeDuration expiryTime) {
       setTimeDuration(properties::setTimeDuration, SLOWNESS_TIMEOUT_KEY, expiryTime);
     }
+
+    String LEADER_LEASE_TIMEOUT_BOUND_RATIO_KEY = PREFIX + ".leader.lease.timeout-bound.ratio";
+    int LEADER_LEASE_TIMEOUT_BOUND_RATIO_DEFAULT = 90;
```

Comment:
- Since this is Read related, let's move it to `RaftServerConfigKeys.Read`. 
- How about calling it `PREFIX + ".leader.lease.timeout.ratio"`?
- Let's use `double` (0 < x <= 1)?